src/utils/contribsUtil/storeAllContribsInDb.js

```diff
@@ -0,0 +1,57 @@
+let fetchRepoContributorsUtil = require('./fetchRepoContribsUtil')
+let model_utils = require('../../models/utils')
+
+// This runs at the very start of page load and stores all the repositories and all the
+// contributors in the database on initial page load
+function storeAllContribsInDb(org) {
+	let AllContributors = [];
+	let promises = [];
+	var contributorSet = new Set([]);
+	return new Promise((resolve, reject) => {
+		model_utils.getItem('allContributors').then((allContributors) => {
+		  // If all contributors list is not in the database, it makes a fresh call to Github API
+		  if(allContributors == null || allContributors == undefined || allContributors.length == 0) {
+			  return model_utils.getItem('repos').then((res) => {
+				let splicedRepos = res.splice(0, 20);
+				splicedRepos.map(function mappingToEachRepo(Repo, i) {
+				  let promise = fetchRepoContributorsUtil.fetchRepoContributorsUtil(org, Repo)
+								.then(function gotRepoContributorsInStorage(contributors) {
+								  if(contributors!=undefined && contributors.length>0) {
+									contributors.map((contributor, i)=> {
+									  if(!contributorSet.has(contributor.login)) {
+										contributorSet.add(contributor.login);
+										AllContributors.push(contributor);
+									  }
```

Comment:
Looks super nice! Some extreme indentation here, though, did you want to adjust that? Thanks!

Reply:
Done!